raster/src/main/scala/geotrellis/raster/io/geotiff/util/ByteReaderExtensions.scala

```diff
@@ -173,33 +173,22 @@ trait ByteReaderExtensions {
     }
 
     final def getSignedByteArray(length: Long, valueOffset: Long): Array[Byte] = {
-      val arr = Array.ofDim[Byte](length.toInt)
-
+      // NOTE: We don't support lengths greater than Int.MaxValue yet (or ever).
```

Comment:
Fixed it. Why it was converted to `Int`? That's because you can't make an `Array.ofDim` using a `Long`.